packages/graphql-toolbox/src/contexts/auth.tsx

```diff
@@ -86,7 +87,8 @@ export function AuthProvider(props: any) {
     [value, setValue] = useState<State>({
         login: async (options: LoginOptions) => {
             const auth = neo4j.auth.basic(options.username, options.password);
-            const driver = neo4j.driver(options.url, auth);
+            const encrypted = options.secure === "on" ? 'ENCRYPTION_ON' : 'ENCRYPTION_OFF';
```

Comment:
Nitpicking of the month: Let's also use double quotes for `ENCRYPTION_ON` and `ENCRYPTION_OFF`

Reply:
You know that I will not survive without your nitpickings :)